platform/broadcom/sonic-platform-modules-dell/s6100/scripts/check-getty.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+RETRY=0
+while [ $RETRY -lt 5 ]; do
+    let RETRY=$RETRY+1
+
+    /bin/systemctl --quiet is-active serial-getty@ttyS1.service
+    status=$?
+    if [ $status == 0 ]; then
+        exit 0
+    fi
+
+    # when serial-getty not running, recheck later, beause systemd will restart serial-getty automatically.
+    sleep 1
```

Comment:
we have seen cases where systemd is not restarting getty and that was the reason in S6100 we had to do it outside systemd like in monit service. Btw, this issue is not specific to S6100